server/views/pages/overview/partials/overviewTabContents.njk

```diff
@@ -35,6 +35,10 @@
           </tbody>
         </table>
 
+        <p class="govuk-body govuk-!-margin-top-2">
```

Comment:
This is the "View all" link at the bottom of the latest functional skills table on the Overview tab